apps/worker/src/app/workflow/usecases/send-message/send-message.command.ts

```diff
@@ -50,4 +51,7 @@ export class SendMessageCommand extends EnvironmentWithUserCommand {
 
   @IsDefined()
   tags: string[];
+
+  @IsOptional()
+  statelessPreferences?: WorkflowChannelPreferences;
```

Comment:
I have opted to take a low-touch approach to adding support for stateless preferences, we will need to refactor the send-message flow at a later point to inject the NotificationTemplate into the SendMessage use-case and downstream use-cases.

This naming reflects this low-touch approach to highlight that stateless Workflows are dealt with separately in the flow.

Reply:
Good idea. I agree 100% with the approach. This should go hand-in-hand with the NotificationTemplate to Workflow remodelling.